downstream/modules/platform/proc-approving-the-imported-collection.adoc

```diff
@@ -2,45 +2,49 @@
 
 = Approving the Imported Collection
 
-
 After you have imported collections with either the GUI or the CLI method, you must approve them by using the GUI. After they are approved, they are available for use.
 
 .Procedure
 
-. Login to private automation hub web console.
+. Login to link:https://console.redhat.com/ansible/automation-hub/[{HubName} Hybrid Cloud Console].
 
 . Go to menu:Collections[Approval].
```

Comment:
This should be changed to:

From the navigation panel, select menu:Collections[Approval].